serverless-offline-elasticmq/types/elasticMQ.d.ts

```diff
@@ -14,6 +14,7 @@ export interface ElasticMQLaunchOptions {
 
 
 export interface ElasticMQConfig {
+  version: string;
```

Comment:
should we make the version optional and default to the latest available at time of merging. that way people aren't forced to set the version themselves, if they don't want to?

Reply:
That's a question I asked myself and honestly, I don't know which option is the best. My first thought was that as there's a breaking change in the AWS SDK API, maybe it's best to force the user to set the version. But as second thought, I guess it's easier to make it optional. So let's do that. I've updated the code to make this field optional.